package.json

```
@@ -20,7 +20,8 @@
   ],
   "author": "JP Richardson <jprichardson@gmail.com>",
   "contributors": [
-    "Alexander J. Lallier <alexanderlallier@aol.com>"
+    "Alexander J. Lallier <alexanderlallier@aol.com>",
+    "Eric Newport <kethinov@gmail.com>"
```

```suggestion-1
"Alexander J. Lallier <alexanderlallier@aol.com>"
```

Eric just want to keep contributors list to added collaborators, this is inline with what we researched around npm with when the project was turned over to me and it helps keep down `package.json` sprawl